Gems/Terrain/Code/Source/TerrainRenderer/TerrainFeatureProcessor.cpp

```diff
@@ -116,44 +116,17 @@ namespace Terrain
 
     void TerrainFeatureProcessor::OnTerrainDataDestroyBegin()
     {
-        m_zBounds = AZ::Vector2::CreateZero();
-        m_dirtyRegion = AZ::Aabb::CreateNull();
+        m_zBounds = {};
     }
     
-    void TerrainFeatureProcessor::OnTerrainDataChanged(const AZ::Aabb& dirtyRegion, TerrainDataChangedMask dataChangedMask)
+    void TerrainFeatureProcessor::OnTerrainDataChanged(const AZ::Aabb& dirtyRegion [[maybe_unused]], TerrainDataChangedMask dataChangedMask)
```

Comment:
Nit: Probably not in the coding standards, but the vast majority of the code has [[maybe_unused]] appearing before the variable, not after.

Reply:
I've seen it used both ways in our code, and I think I'm beginning to prefer after the variable name, seems more readable to me. Fine with putting it in front though if you think standards are going that way.

Reply:
I don't have a preference one way vs the other myself, I just have a general preference for consistency where possible. Most if not all of the uses I've seen in the codebase have it before the variable, which is why it jumped out at me, but I definitely wouldn't block a PR over it.